bottlerocket/testsys/src/run_vmware.rs

```diff
@@ -168,6 +169,15 @@ pub(crate) struct RunVmware {
     /// The arn for the role that should be assumed by the agents.
     #[structopt(long)]
     assume_role: Option<String>,
+
+    /// The path to a TOML file containing custom userdata.
+    #[structopt(long, requires("custom_user_data_mode"))]
+    custom_user_data: Option<String>,
+
+    /// The way custom userdata should interact with the default userdata.
+    /// The possible values are `merge` and `replace`.
+    #[structopt(long, requires("custom_user_data"))]
+    custom_user_data_mode: Option<String>,
```

Comment:
same comment about struct fields not relying on the state of other struct fields being consistent